base/freestanding/femr/docker-compose.yaml

```
@@ -30,6 +33,8 @@ services:
     networks:
       - ingress
       - internal
+    volumes:
+      - ./config/migrations:/opt/migrations
```

minor nitpick: I generally set a bind-mount as read-only, if I know the container doesn't need to write to the host. it's a minor improvement in security, but mostly for preventing unexpected changes eg if you delete files inside a container during testing


```suggestion
      - ./config/migrations:/opt/migrations:ro
```

---

actually, it does need to write there, in the event of using fhir-migrations to generate a migration template script via `flask migrate <name>`